src/commons/mail-service/templates/mail-snippet-overrides.js

```
@@ -72,12 +72,32 @@ const OVERRIDE_TEMPLATE_VARIABLES = Object.freeze([
+function afterSnippetKey(snippetName) {
+  return `${snippetName}${AFTER_SNIPPET_SUFFIX}`;
+}
+
+function isOverridableSnippetKey(name) {
+  if (overridableSnippetSet.has(name)) {
+    return true;
+  }
+  if (
+    typeof name === "string" &&
+    name.endsWith(AFTER_SNIPPET_SUFFIX)
+  ) {
```

ESLint

[error] Replace `⏎····typeof·name·===·"string"·&&⏎····name.endsWith(AFTER_SNIPPET_SUFFIX)⏎··` with `typeof·name·===·"string"·&&·name.endsWith(AFTER_SNIPPET_SUFFIX)`

(prettier/prettier)